### ISSUE TITLE
bug: URI parsing in the admin console results incorrect redirectUris and postLogoutRedirectUris stored

### ISSUE THREAD

**Issue by Fierce-Cat** [logto-io/logto#5748]
### Describe the bug

In the application setting, input fields of Redirect URIs and Post Sign-out Redirect URIs will automatically parse URIs with non-ASCII characters, which will cause the client to fail to pass verification when sending requests to OIDC endpoint with its original redirect URI.

### Expected behavior

Redirect URIs and Post Sign-out Redirect URIs should preserve the original input URI(s).
![msedge_EJxfQEm99D](https://github.com/logto-io/logto/assets/21760321/adbdac97-ffb0-4509-b018-689d0b925ca1)

### How to reproduce?

- Enter URIs with non-English characters in Redirect URIs and Post Sign-out Redirect URIs
- click save
- URIs stored as parsed.
### Context

- [ ] Logto Cloud
- [x] Self-hosted, Logto version = 1.15.0 
  - [x] Container (Docker image)
  - [ ] Raw Node.js

### Temporary solution
Use the Management API to change URIs above.

### Screenshots
Current - after click save:
![msedge_h4B69aULwd](https://github.com/logto-io/logto/assets/21760321/1c244f98-11cf-4ad6-81da-334492d0219f)

Request from client failed:
![image](https://github.com/logto-io/logto/assets/21760321/fab081e3-8d6c-41e3-9131-604a0c368da5)

**Comment by charIeszhao** [comment]
Yeah, this is an acknowledged issue and we're trying to fix that in the next release. Thanks for reporting.
